src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -172,43 +211,36 @@ public static void validateDictionaries( final String name1,
     /**
      * Workhorse routine that takes two dictionaries and returns their compatibility.
      *
-     * This version does not check whether common contigs occur at the same absolute indices in each dictionary.
-     *
      * @param dict1 first sequence dictionary
      * @param dict2 second sequence dictionary
+     * @param checkContigOrdering if true, perform checks related to contig ordering: forbid lexicographically-sorted
+     *                            dictionaries, and require common contigs to be in the same relative order and at the
+     *                            same absolute indices
      * @return A SequenceDictionaryCompatibility enum value describing the compatibility of the two dictionaries
      */
-    public static SequenceDictionaryCompatibility compareDictionaries( final SAMSequenceDictionary dict1, final SAMSequenceDictionary dict2 ) {
-        return compareDictionaries(dict1, dict2, false);
-    }
-
-    /**
-     * Workhorse routine that takes two dictionaries and returns their compatibility.
-     *
-     * @param dict1 first sequence dictionary
-     * @param dict2 second sequence dictionary
-     * @param checkContigIndices if true, check whether the common contigs occur at the same absolute indices
-     * @return A SequenceDictionaryCompatibility enum value describing the compatibility of the two dictionaries
-     */
-    public static SequenceDictionaryCompatibility compareDictionaries( final SAMSequenceDictionary dict1, final SAMSequenceDictionary dict2, final boolean checkContigIndices ) {
-        if ( nonCanonicalHumanContigOrder(dict1) || nonCanonicalHumanContigOrder(dict2) )
+    public static SequenceDictionaryCompatibility compareDictionaries( final SAMSequenceDictionary dict1, final SAMSequenceDictionary dict2, final boolean checkContigOrdering ) {
+        if ( checkContigOrdering && (nonCanonicalHumanContigOrder(dict1) || nonCanonicalHumanContigOrder(dict2)) ) {
             return SequenceDictionaryCompatibility.NON_CANONICAL_HUMAN_ORDER;
+        }
 
         final Set<String> commonContigs = getCommonContigsByName(dict1, dict2);
 
-        if (commonContigs.size() == 0) {
+        if ( commonContigs.size() == 0 ) {
```

Comment:
I'm getting soft.  The only comment I've found so far is that this could be `commonContigs.isEmpty()`